website/docs/r/pool.html.markdown

```diff
@@ -0,0 +1,56 @@
+---
+layout: "libvirt"
+page_title: "Libvirt: libvirt_pool"
+sidebar_current: "docs-libvirt-pool"
+description: |-
+  Manages a storage pool in libvirt
+---
+
+# libvirt\_pool
+
+Manages a storage pool in libvirt. Currently only directory-based storage pool are supported. For more information on
+storage pools in libvirt, see [the official documentation](https://libvirt.org/formatstorage.html).
+
+**WARNING:** This is experimental API and may change in the future.
+
+## Example Usage
+
+```hcl
```

Comment:
feel free to create an example https://github.com/dmacvicar/terraform-provider-libvirt/tree/master/examples

Reply:
Instead of creating a separate example for just pool, I think we should add pool to existing examples as to discourage the assumption of "default" pool's existence. As I'm sure I wrote before, there is no "default" pool guaranteed by libvirt on any of it's daemons. The system daemon usually has this pool available, because virt-manager and virt-install (and other tools) create it.

Reply:
ok we can add the pool creation to the ubuntu.  I can agree with your POV , but we also need to think that each example should showcase a feature and if we add maybe to much new infos, in term of cognition load can be to much.

I'm personally fine with both examples.. Feel free to pick one thx

Reply:
Right. I agree with your POV too and it would be best to keep examples as simple as possible. However, it's pretty annoying for newcomers to not be able to run the examples as is and without pool setup, that will be the experience and IMO that's worse than a bit more of cognitive load.

Reply:
but yeah, for now I'll just add to Ubuntu example. :)

Reply:
sounds good!

Reply:
Done